From 4062482b0add0ffa4644a3dd8da2e0b69af2ada0 Mon Sep 17 00:00:00 2001
From: Suneel Garapati <sgarapati@caviumnetworks.com>
Date: Mon, 11 Sep 2017 15:11:39 -0700
Subject: [PATCH 0458/1200] drivers: ahci: mark message as debug

sff81xx has port multiplier and always shows link up.
So starting ports fail hence mark these to debug messages.

Signed-off-by: Suneel Garapati <sgarapati@caviumnetworks.com>
---
 drivers/ata/ahci.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/ata/ahci.c b/drivers/ata/ahci.c
index 57db8d58b4..c8f3deba1f 100644
--- a/drivers/ata/ahci.c
+++ b/drivers/ata/ahci.c
@@ -1000,7 +1000,7 @@ static int ahci_start_ports(struct ahci_uc_priv *uc_priv)
 	for (i = 0; i < CONFIG_SYS_SCSI_MAX_SCSI_ID; i++) {
 		if (((linkmap >> i) & 0x01)) {
 			if (ahci_port_start(uc_priv, (u8) i)) {
-				printf("Can not start port %d\n", i);
+				debug("AHCI Can not start port %d\n", i);
 				continue;
 			}
 		}
-- 
2.22.0

